aioamqp/__init__.py

```diff
@@ -90,6 +90,9 @@
     if url.scheme not in ('amqp', 'amqps'):
         raise ValueError('Invalid protocol %s, valid protocols are amqp or amqps' % url.scheme)
 
+    if url.scheme == 'amqps' and not kwargs.get('ssl'):
+        kwargs['ssl'] = True
```

Comment:
would it be acceptable, if I change it to `ssl.create_default_context()`

Reply:
yes !